propertysuggester/test/parser/test_csv_writer.py

```diff
@@ -29,16 +29,16 @@ def test_write_csv(self):
         out.seek(0)
 
         line = out.readline()
-        eq_('Q51,claim,31,wikibase-entityid,Q5107', line.strip())
+        eq_('Q51,claim,31,wikibase-entityid,Q5107', str(line.strip()))
```

Comment:
How can a strip function return something that's not already a string?

Reply:
Same as above

Reply:
Same as what? There is a `byte` type in Python? What does it mean to "strip" a sequence of bytes? This sounds very much suspicious.

Reply:
yeah, fixed it.